lib/cpp/linear_model/model_logreg.cpp

```diff
@@ -6,66 +6,75 @@
 
 #include "tick/linear_model/model_logreg.h"
 
-ModelLogReg::ModelLogReg(const SBaseArrayDouble2dPtr features,
-                         const SArrayDoublePtr labels,
-                         const bool fit_intercept,
-                         const int n_threads)
-    : ModelGeneralizedLinear(features, labels, fit_intercept, n_threads),
-      ModelLipschitz() {}
-
-const char *ModelLogReg::get_class_name() const {
```

Comment:
get_class_name has disappeared

Reply:
it's now a getter 

https://github.com/Dekken/tick/blob/template_linear_model/lib/include/tick/linear_model/model_logreg.h#L39